### ISSUE TITLE
MacBook’s Dota2 has no respond after updating 

### ISSUE THREAD

**Issue by Realkai1221** [ValveSoftware/Dota2-Gameplay#2606]
### Description

After updating for battle pass my Mac cannot load the game. Only black screen and beach ball. Tried reinstalling the game but still the same turns out.

### Screenshots

![image](https://user-images.githubusercontent.com/112666364/188028202-4fb27354-0265-4af2-8626-460059ac2c8d.jpg)
![image](https://user-images.githubusercontent.com/112666364/188028224-24d8bb80-18a1-4a94-a28b-29db7605891d.jpg)
![image](https://user-images.githubusercontent.com/112666364/188028250-565c8e3c-fdfd-4cc8-a16d-0f683d30e57d.jpg)

**Comment by fajaranshari** [comment]
Same issue with MacOS 12.3.1 Monterey
<img width="583" alt="image" src="https://user-images.githubusercontent.com/19646168/188031276-fd31e9ec-b032-4dc7-9c88-639187ac4788.png">

**Comment by Wispoi** [comment]
same issue

**Comment by exegeses** [comment]
same issue here

**Comment by nich961996** [comment]
Same issue

**Comment by MohmmadSawalha-ios** [comment]
same issue

**Comment by bratanyu** [comment]
same issue

**Comment by fveracoechea** [comment]
Same issue here...
![image](https://user-images.githubusercontent.com/26258895/188039023-2dc96ab5-55ae-4a75-9e85-6755f8c66a6b.png)

has anyone tried re-installing the game?

**Comment by MohmmadSawalha-ios** [comment]
> Same issue here... ![image](https://user-images.githubusercontent.com/26258895/188039023-2dc96ab5-55ae-4a75-9e85-6755f8c66a6b.png)
> 
> has anyone tried re-installing the game?

I did, nothing same issue :(

**Comment by maicojay16** [comment]
Same issue here, Im using MacBook Pro 2018.

![Screen Shot 2022-09-02 at 9 42 05 AM](https://user-images.githubusercontent.com/23121374/188041376-34a21c09-7008-45df-ae8d-0361eb95c6dd.png)

Before the update everything works fine, I even play 1 game with my friend. After i update the new patch for Battlepass 2022, Im stuck in the opening loading screen of Dota.

I tried to reinstall it but nothing works. Please help!!!

**Comment by Mzuha** [comment]
Same thing, reinstalling didn't help

**Comment by MohmmadSawalha-ios** [comment]
@jeffhill  can you please advise on this ? are we expecting a fix soon so we can enjoy the game again ?

**Comment by nikelsk8** [comment]
same issue...

**Comment by forrsaken** [comment]
same issue here, stuck on black screen when opening Dota2 after BattlePass update

**Comment by ronakparhi** [comment]
same issue here :(

**Comment by HiroAsashi** [comment]
Same issue on Mac OS 11.6.8

**Comment by bredikhin13** [comment]
Same issue, MacBook Pro, macOS 10.15.7

**Comment by RoeelCohen** [comment]
same

**Comment by JeffreyCheong** [comment]
I am surprised they haven't fix this after 15 hours

**Comment by MohmmadSawalha-ios** [comment]
almost 24 hours since the update still no response the engineers are distributing localization patches and ignoring a full platform OS :)))))) i think we should start think about changing moba game absolutely joke of a company

**Comment by Realkai11** [comment]
There was just an update but i still cannot load dota2 on mac

**Comment by MohmmadSawalha-ios** [comment]
yes updating ui issues with the new cosmetics have higher priority 

> There was just an update but i still cannot load dota2 on mac
